packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -637,4 +638,9 @@ export class ProjectActions {
       }
     }
   }
+
+  async debugCloudRun ({ runNumber }: { runNumber: number }) {
```

Comment:
Why not just accept `runNumber` as a param - object wrapper seems unnecessary

Reply:
Updated.